### PR TITLE
bugfix: conditional rendering

### PR DESCRIPTION
**Pull request by jsJunky**
👍

### CODE REVIEW

**Comment by jsJunky** [comment]
@robrollner Thoughts on this? Should I close it out?

File: weather/src/components/WeatherContainer.js [hunk 1]
```diff
@@ -78,10 +72,11 @@ class WeatherContainer extends Component {
           </Col>
         </Row>
           <Row>
-              <WeekForecast
+            {this.state.result.list ? <WeekForecast
```
Comment (jsJunky) [inline weather/src/components/WeatherContainer.js:75]: Doing an inline conditional but this could be brought to a function instead.

**Review by jsJunky — COMMENTED** [review]
Let me know if you're ok with the inline or if you'd prefer it be a function.

File: weather/src/components/WeatherContainer.js (hunk 1 above)
Comment (robrollner) [reply weather/src/components/WeatherContainer.js:75]: Thanks again for the help Ron! Sorry I had that open, it's been hectic

**Pull request merged**